app/javascript/angular/code/controllers/_sessions_list_ctrl.js

```diff
@@ -153,8 +126,9 @@ export const SessionsListCtrl = (
       });
 
       elmApp.ports.loadMoreSessions.subscribe(() => {
-        $scope.updateSessionsPage();
-        $scope.$apply();
+        sessions.fetch({
+          fetchedSessionsCount: params.get("fetchedSessionsCount")
```

Comment:
Could we do something like `sessions.length` in the sessions module and not use this at all?